Documentation/ApiOverview/LinkHandling/LinkBuilder.rst

```diff
@@ -0,0 +1,35 @@
+.. include:: /Includes.rst.txt
+.. index:: Link builder
+.. _link-builder:
+
+=====================
+Frontend link builder
+=====================
+
+A link builder, a class extending the abstract class
+:php:`\TYPO3\CMS\Frontend\Typolink\AbstractTypolinkBuilder` is called whenever
+a link is rendered in the frontend.
+
+There are specific link builders for each type of link. Which link is to
+be called is determined by the class configured in global configuration:
+:php:`$GLOBALS['TYPO3_CONF_VARS']['FE']['typolinkBuilder'][<type>]`.
```

Comment:
Can we use a code snippet and add the additional.php caption to it?

Reply:
This would be better done in the ext_localconf.php of the extension providing the link builder. Added it.